src/flux/course.coffee

```diff
@@ -0,0 +1,46 @@
+_ = require 'underscore'
+
+{TaskActions, TaskStore} = require './task'
+{CrudConfig, makeSimpleStore, extendConfig} = require './helpers'
+
+CourseConfig =
+
+  createPractice: (courseId) ->
+    # @create(@freshLocalId())
+    # Contemplating how to use store with local id.
+    @_local[courseId] ?= {}
+
+  createdPractice: (obj, courseId) ->
+    @emit('practice.created', obj.id)
+    @loadedPractice(obj, courseId)
+
+  loadPractice: (courseId) ->
+    @load(courseId, {})
+
+  loadedPractice: (obj, courseId) ->
+    @loaded({practice: obj}, courseId)
+    obj.type = 'practice'
+
+    TaskActions.loaded(obj, obj.id)
+    @emit('practice.loaded', obj.id)
+
+
+  exports:
+    getPracticeId: (courseId) ->
+      @_get(courseId)?.practice?.id
+
+    hasPractice: (courseId) ->
+      @_get(courseId)?.practice?
+
+    getPractice: (courseId) ->
+      if @_get(courseId)?.practice?
+        task = TaskStore.get(@_get(courseId)?.practice?.id)
+        @emit('practice.loaded', task.id)
```

Comment:
Is this for a task that already has an existing practice? If so, could this be emitted in `_loaded: (obj, id) ->` instead of in a getter?